QModManager/API/QModHooks.cs

```diff
@@ -54,11 +54,11 @@ internal static void Load()
         {
             SceneManager.sceneLoaded += (scene, loadSceneMode) => SceneLoaded?.Invoke(scene, loadSceneMode);
 
-#pragma warning disable CS0618 // Type or member is obsolete
-            Hooks.Load();
+//#pragma warning disable CS0618 // Type or member is obsolete
+//            Hooks.Load();
```

Comment:
Why disable hooks?

Reply:
After removing the Mod API classes these references were broken too. 
Commented then out just to get things compiling again.